### DISCUSSION TITLE
Documentation of how to run podman with systemd-run

### DISCUSSION THREAD

**Discussion by eriksjolund** [containers/podman#18800] [Show and tell]
I started writing some documentation of how to run __podman__ with __systemd-run__
https://github.com/eriksjolund/podman-systemd-run-docs

It's currently a work-in-progress and rather experimental because I have just a sketchy understanding of how things works.

**Comment by rhatdan** [comment]
Have you tried.

machinectl shell user1@ podman info

**Reply by eriksjolund** [reply]
Interesting, I didn't know it was possible to put a command in the end of the line.
I tried it out and it works. 

But I think the missing exit code functionality makes it less useful than `systemd-run`

Quote from `man machinectl` 
_"Note that machinectl shell does not propagate the exit code/status of the invoked shell process."_
